src/lib.rs

```diff
@@ -5,5 +5,7 @@
 //! It is intended to be used as an unopinionated base for building higher level data structures
 //! representing traces/tasks/etc..
 
+#[cfg(test)] #[macro_use] extern crate approx;
```

Comment:
```suggestion
#[cfg(test)] 
#[macro_use]
extern crate approx;
```
Iirc rustfmt will put attributes on separate lines by default.

Reply:
after #12 is merge this will actually not be needed anymore at all 😉 

also I have a PR queued up after #10 that applies `rustfmt` and adds a CI job to keep it always formatted correctly

Reply:
Fair enough